api/BarcodeDetector.json

```
@@ -6,7 +6,8 @@
         "spec_url": "https://wicg.github.io/shape-detection-api/#barcode-detection-api",
         "support": {
           "chrome": {
-            "version_added": "83"
+            "version_added": "83",
+            "notes": "Only Android and macOS, see <a href='https://www.chromestatus.com/feature/4757990523535360'>feature 4757990523535360</a>."
```

Chrome for Android (and WebView) are represented as separate browsers, below, so there's no need to mention Android in the desktop entry.

Following https://github.com/mdn/browser-compat-data/issues/7184#issuecomment-775314742, this would probably be best represented here (and throughout) as:

```suggestion
            "version_added": "83",
            "partial_implementation": true,
            "notes": "Supported on macOS only."
```